tests/test_parser.py

```diff
@@ -228,6 +228,27 @@ def test_modify_instruction(self, dfparser):
         dfparser.cmd = CMD[1]
         assert dfparser.cmd == CMD[1]
 
+    def test_modify_from_multistage(self, dfparser):
+        FROM = ('base:${CODE_VERSION}', 'extras:${CODE_VERSION}')
+        CMD = ('/code/run-app', '/code/run-extras')
+        df_content = dedent("""\
+ARG  CODE_VERSION=latest
+FROM {0}
+CMD  {1}
+
+FROM {2}
+CMD  {3}""").format(FROM[0], CMD[0], FROM[1], CMD[1])
+
+        dfparser.content = df_content
+
+        assert dfparser.baseimage == FROM[0]
+        dfparser.baseimage = FROM[1]
+        assert dfparser.baseimage == FROM[1]
```

Comment:
this doesn't actually test that the second 'from' wasn't modified, as getter is returning 1st, but we aren't checking value of 2nd 'from'